bin/preview.sh

```diff
@@ -15,6 +15,12 @@ fi
 
 IFS=':' read -r -a INPUT <<< "$1"
 FILE=${INPUT[0]}
+
+if [[ $1 =~ ^[A-Z]:\\ ]]; then
+  FILE=$FILE:${INPUT[1]}
+  shift
+fi
```

Comment:
Shouldn't we also change the value of `$CENTER`, to maybe `INPUT[2]`? And if I'm not mistaken, `shift` is not needed here.

Reply:
My intent is to shift `INPUT` to avoid additional changes and using `else` but I placed if statement in the wrong spot.